pkg/clone/replicate.go

```diff
@@ -252,12 +252,12 @@ func (r *Replicator) run(ctx context.Context) error {
 
 	g, ctx := errgroup.WithContext(ctx)
 
-	transactions := make(chan Transaction)
+	transactions := make(chan Transaction, r.config.ReplicationParallelism)
```

Comment:
we typically pass in `--replication-parallelism = 8` or `--replication-parallelism = 16`

Reply:
Parallelism